src/routes/index.svelte

```diff
@@ -1,23 +1,19 @@
 <script lang="ts">
-	import { onMount } from 'svelte';
 	import HelloWorld from '../components/HelloWorld.svelte';
 
-	let FlagProvider;
+	import '../global.css';
 
-	onMount(async () => {
-		const proxyClientSvelte = await import('$lib');
-		({ FlagProvider } = proxyClientSvelte);
-	});
+	import { FlagProvider } from '$lib';
 
 	const config = {
-		url: 'https://unleash-proxy.nunogois.com/proxy',
-		clientKey: 'ng-unleash-secret',
+		url: 'https://app.unleash-hosted.com/demo/api/frontend',
+		clientKey:
+			'proxy-client-svelte:development.f7d5c2f3633769afea4eadecd6bf46194dccec9c75170930c76ab4f0',
```

Comment:
is this intentional?

Reply:
Yes, it's a frontend token pointing to our demo instance frontend API, limited to a specific project and environment. I see no harm in it.